R/intervals.r

```diff
@@ -217,7 +217,10 @@ interval <- function(start, end = NULL, tzone = tz(start)) {
 }
 
 parse_interval <- function(x, tz) {
-  mat <- str_split_fixed(x, "/", 2)
+  mat <- matrix(    
+    c( gsub( '(^[^/]+)/(.+$)', '\\1', x ), gsub( '(^[^/]+)/(.+$)', '\\2', x ) ),
```

Comment:
you please add a comment of what this does?

Reply:
Ah, missed one! I've added a comment and removed str_extract, and updated the commit title. Thanks for the feedback.